pkg/artifactory/resource/repository/federated/resource_artifactory_federated_repository_test.go

```diff
@@ -100,6 +101,9 @@ func federatedTestCase(repoType string, t *testing.T) (*testing.T, resource.Test
 		"xrayIndex":    xrayIndex,
 		"memberUrl":    federatedMemberUrl,
 	}
+
+	repoTypeAdjusted := local.GetPackageType(repoType)
+
```

Comment:
What if the repoType is not one of the supported types ? Any special handling to indicate Not a supported type ?

Reply:
The verification for the not supported types is in the schema. The provider won't take the HCL with the type, which is not in the list of supported repos. It's not done in the test here.

Reply:
ok